script.skinshortcuts/default.py

```diff
@@ -307,11 +304,22 @@ def _manage_shortcuts( self, group, defaultGroup, nolabels, groupname ):
         if homeWindow.getProperty( "skinshortcuts-loading" ) and int( calendar.timegm( gmtime() ) ) - int( homeWindow.getProperty( "skinshortcuts-loading" ) ) <= 5:
             return
 
+        # Get current Super Favourites context menu status
+        sfMenu = None
+        if xbmc.getCondVisibility( "System.HasAddon(plugin.program.super.favourites)" ):
```

Comment:
The setting is only changed whilst the scripts dialog is open - it's put back to the users choice afterwards.

It's done - [at Spoysers suggestion](http://forum.kodi.tv/showthread.php?tid=192662&pid=2353547#pid2353547) - because when this setting is enabled in super favourites attempts by the user to trigger the context menu don't trigger the normal id's that can be checked for in onAction. Instead, Super Favourites own context menu shows - which also closes this scripts window.